### DISCUSSION TITLE
Wrong Walletadress

### DISCUSSION THREAD

**Discussion by Danchen1983** [blockscout/blockscout#10416] [General]
Hi,
I wanted to get the new BTC20 Smart Chain on MetaMask. Unfortunately, I took the wrong Walletadress (I wanted to use the Walletadress from WAAS, but I used the Walletadress from Boomerang) and now I can't change anymore my walletadress. Always when I login , I am loged in with my boomerang adress. How can I delete my boomerang adress and put in my WAAS adress?  Thank you very much for helping

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
